### PR TITLE
New xSPWebAppSiteUseAndDeletion resource

### PR DESCRIPTION
**Pull request by ykuijs**
New xSPWebAppSiteUseAndDeletion resource, as mentioned in issue #112. Please review and let me know your comments.

Includes a small update in the mof file of the Blocked File Types resource

### CODE REVIEW

File: Modules/xSharePoint/DSCResources/MSFT_xSPWebAppSiteUseAndDeletion/MSFT_xSPWebAppSiteUseAndDeletion.schema.mof [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+[ClassVersion("1.0.0"), FriendlyName("xSPWebAppSiteUseAndDeletion")]
+Class xSPWebAppSiteUseAndDeletion : OMI_BaseResource
+{
+    [Key] string Url;
+    [write] boolean SendUnusedSiteCollectionNotifications;
+    [write] int32 UnusedSiteNotificationPeriod;
+    [write] boolean AutomaticallyDeleteUnusedSiteCollections;
+    [write] int32 UnusedSiteNotificationsBeforeDeletion;
+    [Write, EmbeddedInstance("MSFT_Credential")] string InstallAccount;
+};
```
Comment (BrianFarnhill) [inline Modules/xSharePoint/DSCResources/MSFT_xSPWebAppSiteUseAndDeletion/MSFT_xSPWebAppSiteUseAndDeletion.schema.mof]: Need a newline at the end of the file here

**Comment by BrianFarnhill** [comment]
Can I also get you to update the readme file (you'll see it at the root of the directory, or under the "documentation" solution folder if you open in visual studio) and just add the name of this resource to the "unreleased" section of the version history. We are starting to do that with these projects now to help track changes made between releases. Other than that though it looks all good buddy.

**Comment by ykuijs** [comment]
Sure, will do that!

**Comment by ykuijs** [comment]
Changes done. Please review!

**Comment by BrianFarnhill** [comment]
Thanks dude, merging in now

**Pull request merged**